src/index.ts

```diff
@@ -287,18 +303,24 @@ class MagicNixCacheAction {
     const pid = parseInt(await fs.readFile(pidFile, { encoding: "ascii" }));
     actionsCore.debug(`found daemon pid: ${pid}`);
     if (!pid) {
-      throw new Error("magic-nix-cache did not start successfully");
+      const msg = "magic-nix-cache did not start successfully";
+      this.failInStrictMode(msg);
+      throw new Error(msg);
     }
 
     const log = tailLog(this.daemonDir);
 
     try {
       actionsCore.debug(`about to post to localhost`);
-      const hostAndPort = inputs.getString("listen");
-      const res: Response = await this.client
-        .post(`http://${hostAndPort}/api/workflow-finish`)
-        .json();
-      actionsCore.debug(`back from post: ${res}`);
```

Comment:
You kept the `debug` log in the above similar snippet but missed it here, probably want to bring it back in debug mode for people who don't have `strict-mode` enabled....... :sweat_smile:

Reply:
Hehe, good call